woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -522,6 +524,64 @@ public static function get_edit_order_url( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Returns the admin screen id for orders.
+	 *
+	 * This method detects the screen_id according to HPOS availability,
+	 * as cannot rely anymore on the shop_order registered post type as the screen ID.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return string
+	 */
+	public static function get_screen_id( ) {
+
+		return SV_WC_Plugin_Compatibility::is_hpos_enabled() ? wc_get_page_screen_id( 'shop-order' ) : 'shop_order';
+	}
+
+
+	/**
+	 * Determines whether the given $post_id is a WooCommerce order or not according to HPOS availability.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param int $post_id post ID.
+	 * @return bool
+	 */
+	public static function is_order( $post_id ) {
+
+		if ( ! SV_WC_Plugin_Compatibility::is_hpos_enabled() ) {
```

Comment:
same here, pending for this method to be added

Reply:
@unfulvio yes, it's related to [MWC-10118](https://godaddy-corp.atlassian.net/browse/MWC-10118), so I added another story here.

It's coming from the 'Methods to upgrade' section in the [Discovery Doc](https://docs.google.com/document/d/1752eyK4-9d-bxlOc2ZbAbZqEk91O5aPPrZHC-U0ovYE/edit#).